--- origsrc/glib2-1.1.4/ext/glib2/rbglib.h	2012-08-22 19:08:24.839185100 -0500
+++ src/glib2-1.1.4/ext/glib2/rbglib.h	2012-08-22 19:09:09.809757200 -0500
@@ -116,7 +116,7 @@ typedef int GPid;
 #define G_DEF_ERROR2(domain, name, module, parent) \
     rbgerr_define_gerror(domain, name, module, parent, Qnil)
 
-#if defined(G_PLATFORM_WIN32) && !defined(RUBY_GLIB2_STATIC_COMPILATION)
+#if defined(G_OS_WIN32) && !defined(RUBY_GLIB2_STATIC_COMPILATION)
 #  ifdef RUBY_GLIB2_COMPILATION
 #    define RUBY_GLIB2_VAR __declspec(dllexport)
 #  else
